### ISSUE TITLE
Struggling with pluralization syntax

### ISSUE THREAD

**Issue by benwrigley** [cyberwolf-studio/lingua#4]
Hey, sorry me again!

I've just trying to get pluralization working and not sure I'm using the right syntax:

in my Vue component:

```
trans('feed.postCommentCount',{'count' : post.descendants_count}) 
```

in my lang file:
```
    'postCommentCount' => ':count comment|:count comments',
```

output is:
`
1 comment|:value comments
`
or
`
8 comment|:value comments
`

Looing at the `trans()` function, do I need a third parameter as a Boolean?
```
trans('feed.postCommentCount',{'count' : post.descendants_count}, true)
```

Thanks!

**Comment by mikield** [comment]
Yes, but I think we can provide a `trans_choice` function that will already have `shouldPluralize` flag set to `true`

**Comment by mikield** [comment]
Also, I need to check if that kind of pluralization will work.

I have created test for `{NUMER} TRANS | {NUMER} TRANS` syntax, but have not thought about simple one-many syntax.

**Comment by benwrigley** [comment]
Ah ok, I was going to say that I'm still getting same output even with third argument.

I'll try reformatting my lang file...

**Comment by benwrigley** [comment]
ah brilliant!

so this works:

```
transChoice('feed.postCommentCount',{'count' : post.descendants_count}
```

```
'postCommentCount' => '{1}:count comment|[2,*]:count comments',
```

but just in case you wanted to know. This doesn't work atm:

```
'postCommentCount' => ':count comment|:count comments',
```

Thanks again for all you help and the brilliant package!
